cmd/azure.go

```diff
@@ -131,12 +147,7 @@ func (a AzureObjects) GetObject(bucket, object string, startOffset int64, length
 	}
 	rc, err := a.client.GetBlobRange(bucket, object, byteRange, nil)
 	if err != nil {
-		if e, ok := err.(storage.AzureStorageServiceError); ok {
-			if e.StatusCode == http.StatusNotFound {
-				return traceError(ObjectNotFound{bucket, object})
-			}
-		}
-		return traceError(err)
+		return traceError(azureToObjectError(err, bucket, object))
```

Comment:
it will not matter because traceError captures the line number in either case.

if it's about consistency, we can follow `azureToObjectError(traceError(err), bucket, object)?`